app/views/account/renew.scala.html

```diff
@@ -5,13 +5,29 @@
 @import com.gu.salesforce.Contact
 @import views.support.Dates.prettyDate
 @import com.gu.i18n.Country
+@import controllers.ManageWeekly.WeeklyPlanInfo
 
 @(
-    subscription: Subscription[SubscriptionPlan.WeeklyPlan], billingSchedule: BillingSchedule, contact: Contact, billToCountry: Country
+    subscription: Subscription[SubscriptionPlan.WeeklyPlan], billingSchedule: BillingSchedule, contact: Contact, billToCountry: Country, plans: List[WeeklyPlanInfo]
 )(implicit r: RequestHeader, touchpointBackendResolution: services.TouchpointBackend.Resolution)
 
 @main("Your Guardian Weekly subscription | The Guardian", bodyClasses = List("is-wide"), edition = UK, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
 
+    <script>
+
+        guardian.plans = [
+            @plans.map { plan =>
+                {
+                    id: '@plan.id.get'
+                    price: '@plan.price.prettyAmount'
+                    currency: '@plan.price.currency'
+                    billingPeriod: '@plan.billingPeriod.noun'
+                },
```

Comment:
Yeah, it should be alright. I'll figure out what I need when I've got the promo stuff done.